utils/rpms/debian/cart.install

```diff
@@ -1,7 +1,5 @@
 usr/bin/cart_ctl
-usr/bin/crt_launch
 usr/bin/*-cart_ctl
-usr/bin/*-crt_launch
```

Comment:
These two should remain removed from the RPM?

Reply:
I don't build them on non-centos platforms.  My plan was to get testing working and then we can workout how to get the right openmpi installed on those platforms.